src/tree/txTree.ts

```
@@ -0,0 +1,269 @@
+        if (txid === this.txid) {
+            fn(this.root);
+            return;
+        }
+
+        for (const child of this.children.values()) {
+            try {
+                child.update(txid, fn);
+                return;
+            } catch (error) {
+                // Continue searching in other children if not found
+                continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)